CougSat1-Ground/src/space/cougs/ground/utils/FileUtils.java

```diff
@@ -0,0 +1,19 @@
+package space.cougs.ground.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+
+public class FileUtils {
+
+	public static final long readNextBytes(FileInputStream file, int numberOfBytes) throws IOException {
+		long buff = 0;
+
+		for (int i = 0; i < numberOfBytes; i++) {
```

Comment:
while (numberOfBytes > 0){
numberOfBytes--
......
Less memory than the for loop